dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp

```diff
@@ -168,7 +168,7 @@ std::tuple<std::optional<RegionRetryList>, RegionException::RegionReadStatus> Ma
                         throw TiFlashException(
                             Errors::Coprocessor::BadRequest,
                             "Income key ranges is illegal, region_id={} version={} conf_version={} request_range=[{}, "
-                            "{}) region_range=[{}, {}}",
+                            "{}) region_range=[{}, {}]",
```

Comment:
We need to backport this to release-7.5 and release-8.1
https://github.com/pingcap/tiflash/blob/release-7.5/dbms/src/Flash/Coprocessor/DAGStorageInterpreter.cpp#L152-L157